### PR TITLE
API implementation done for Netsuite Export Reference Number

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

**Comment by ruuushhh** [comment]
image (1)
image
For Bill

File: apps/netsuite/connector.py [hunk 1]
```diff
@@ -1145,6 +1145,7 @@ def __construct_credit_card_charge(
             },
             'tranDate': transaction_date,
             'memo': credit_card_charge.memo,
+            'tranid': credit_card_charge.reference_number,
```
Comment (NileshPant1999) [inline apps/netsuite/connector.py:1148]: are you sure this is not camelcase ??

File: apps/netsuite/models.py [hunk 2]
```diff
@@ -274,6 +274,17 @@ def get_ccc_account_id(configuration: Configuration, general_mappings: GeneralMa
 
     return ccc_account_id
 
+def get_report_or_expense_number(expense_group: ExpenseGroup) -> str:       
+        expense: Expense = expense_group.expenses.first()
+        expense_group_settings: ExpenseGroupSettings = ExpenseGroupSettings.objects.get(workspace_id= expense_group.workspace_id)
+        if expense_group.fund_source == "CCC":
```
Comment (ashwin1111) [inline apps/netsuite/models.py:280]: use single quotes for string, just for consistency

**Review by ashwin1111 — APPROVED** [review]

File: apps/netsuite/connector.py (hunk 1 above)
Comment (ruuushhh) [reply apps/netsuite/connector.py:1148]: We have to send it in this format to netsuit

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**